opentelemetry-stdout/src/logs/exporter.rs

```diff
@@ -44,108 +30,77 @@ impl fmt::Debug for LogExporter {
 
 #[async_trait]
 impl opentelemetry_sdk::export::logs::LogExporter for LogExporter {
-    /// Export spans to stdout
+    /// Write logs to stdout
     async fn export<'a>(&mut self, batch: Vec<Cow<'a, LogData>>) -> ExportResult {
-        if let Some(writer) = &mut self.writer {
-            // TODO - Avoid cloning logdata if it is borrowed.
-            let log_data = crate::logs::transform::LogData::from((
-                batch.into_iter().map(Cow::into_owned).collect(),
-                &self.resource,
-            ));
-            let result = (self.encoder)(writer, log_data) as LogResult<()>;
-            result.and_then(|_| writer.write_all(b"\n").map_err(|e| Error(e).into()))
+        if self.is_shutdown.load(atomic::Ordering::SeqCst) {
+            return Err("exporter is shut down".into());
         } else {
-            Err("exporter is shut down".into())
+            if self.resource_emitted {
+                print_logs(batch);
+            } else {
+                self.resource_emitted = true;
+                println!("Resource");
+                if self.resource.schema_url().is_some() {
```

Comment:
You could simplify this a bit and avoid calling `resource.schema_url()` twice.

```rust
if let Some(schema_url) = self.resource.schema_url() {
    println!("\t Resource SchemaUrl: {:?}", schema_url);
}
``

Reply:
Did you miss this one?

Reply:
oops. some refactoring while conflict resolution might have messed up. will fix